docs/guides/platform/get-started/getting-started-with-linode/index.md

```diff
@@ -45,12 +45,13 @@ Newly created Linode accounts have restrictions on ports `25`, `465`, and `587`.
 1.  Sign in and enter your billing and account information. Most accounts are activated instantly, but some require manual review prior to activation. If your account is not immediately activated, you will receive an email with additional instructions.
 
 ## Create a Linode
-![Create a Linode by selecting the image, the region, the plan, and creating a label and a password.](getting-started.gif)
 
 1.  Log in to the [Cloud Manager](https://cloud.linode.com) with the username and password you created when signing up.
 
 1.  At the top of the page, click **Create** and select **Linode**.
 
+![Getting Started Add Linode](getting-started-add-linode.png "Getting Started with Linode")
+
```

Comment:
Please indent the image because the steps are getting renumbered:https://deploy-preview-3873--nostalgic-ptolemy-b01ab8.netlify.app/docs/guides/getting-started/